docs/manual/working/scalaGuide/main/tests/ScalaTestingWithComponents.md

```diff
@@ -0,0 +1,47 @@
+<!--- Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com> -->
+# Testing without Guice
+If you're manually wiring up your application or using [[compile time dependency injection|ScalaCompileTimeDependencyInjection]] then you can directly use and customise your application components, or create a test variant specific to your test case. You can also modify and override filters, define routes, and specify configuration.
+
+## BuiltInComponentsFromContext
+[BuiltInComponentsFromContext](api/scala/play/api/BuiltInComponentsFromContext.html)  gives us an easy way to bootstrap your components. Given the context, this provides all required built in components: `environment`, `configuration`, `applicationLifecycle`, etc.
+
+As described in [[compile time dependency injection|ScalaCompileTimeDependencyInjection]], this is the most common way of wiring up the application manually.
+
+When testing, we can use the real components which allows us to start the complete application for full functional testing, or we can create a test components which starts a subset of the application as required.
+
+## WithApplicationComponents
+Key to testing the components is the [WithApplicationComponents](api/scala/org/scalatestplus/play/components/WithApplicationComponents.html) trait. This sets up the application, server and context ready for testing. Similar to when [[Testing with Guice|ScalaTestingWithGuice]], there are a number of `sub-traits` available to mixin depending on your testing strategy
```

Comment:
I would avoid citing Guice anywhere here. So you can rewrite from:

> Similar to when testing with Guice, there are a number of...

To only have:

> There are a number of...